packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -120,6 +121,11 @@ class RefreshIndicator extends StatefulWidget {
   /// its actual displacement may significantly exceed this value.
   final double displacement;
 
+  /// The distance of [RefreshProgressIndicator] from the edge of the screen.
```

Comment:
Can you clarify what this means?  Is it the distance when the indicator is visible, or the distance during a specific point in animating it?

It seems like this is also the additional distance beyond what would be default - is that so? If so, it would be helpful to document it.

Reply:
I tried explaining in the PR's description... This controls how far the spinner is from the edge of the screen. Previously it's always equal to 0, which means that if you "pull to refresh" the indicator will always start showing from the edge of the screen. This may not always be the case.

Reply:
Great, thanks!